packages/parts/src/lib/maps/mapbox/map/Map.svelte

```diff
@@ -7,14 +7,15 @@
   import type { LngLatLike, MapboxOptions, Map, LngLat, ErrorEvent, EventData } from 'mapbox-gl';
   import { bindEvents } from '../event-bindings';
   import { getTimeZoneLongitude } from '../../utils/getTimeZoneLongitude';
+  import { ADDED_FEATURE_ID_PREFIX } from '../../utils/randomId';
   import { PUBLIC_mapboxAccessToken } from '$env/static/public';
 
   export let map: Map = null;
   export let version = 'v2.9.2';
   export let customStylesheetUrl: string = undefined;
   export let accessToken = PUBLIC_mapboxAccessToken;
   export let options: Partial<MapboxOptions> = {};
-  export let zoom = 2;
+  export let zoom = 4;
```

Comment:
Next time I would prefer we don't change the base component default, but I guess I'll let this change go through this time since you manually entered 2 in the only map usage that occurs where we don't want this change.